website/addons/osfstorage/model.py

```diff
@@ -218,6 +241,20 @@ def path(self):
     def node(self):
         return self.node_settings.owner
 
+    @property
+    def rented(self):
```

Comment:
I don't think the difference between `node.rented` and `node.renter._id` justifies the existence of a property method.

That said, it's just my opinion and not a bug - so I defer to your judgment.